tests/common/fixtures/conn_graph_facts.py

```diff
@@ -14,14 +19,16 @@ def fanout_graph_facts(localhost, duthost, conn_graph_facts):
     for intf in conn_graph_facts["device_conn"]:
         fanout = conn_graph_facts["device_conn"][intf]["peerdevice"]
         if fanout not in facts:
-            facts[fanout] = get_graph_facts(duthost, localhost, fanout)
+            facts[fanout] = get_graph_facts(duthost, localhost,
+                                            hostname=fanout)
     return facts
 
 
-def get_graph_facts(duthost, localhost, host_name):
+def get_graph_facts(duthost, localhost, hostname=None, hostnames=None):
```

Comment:
instead of multiple args (hostname, hostnames), you could just pass in a list(for multiple duts) or a string(single dut) and let this method differentiate between both using isinstance

Reply:
That's a viable solution, I will modify then.